### DISCUSSION TITLE
NASA STELLA-Q2 spectrometer data readings for vegetative species end members to be used for spectral unmixing and mapping of such features.

### DISCUSSION THREAD

**Discussion by Philliec459** [opengeos/HyperCoast#33] [Ideas]
The Sarasota Science and Technology Society (STS) has been working with NASA on Landsat interpretations and enhancement of these interpretations using NASA's STELLA hand held spectrometers that provide 18 spectral channels of data in the visual and NIR portions of the spectrum. This is all a part of our community involvement and STEM initiatives. A brief overview is provided in our GitHub repository(https://github.com/Philliec459/Science-and-Technology-Society-Use-of-NASA-STELLA-Q2-Spectrometer).

We are already using our spectral vegetative species libraries to aid in our Landsat interpretations where NDVI calculations are quite helpful, but in the coming few weeks we will be meeting with NASA to discuss  possibly ways to use drone conveyed STELLA instruments to enhance PACE Red Tide interpretations. We welcome any ideas on this subject and how we might possibly use HyperCoast to aid in these enhancements and interpretation products.

**Comment by bingqing-liu** [comment]
Hi Craig,

Thank you for connecting us! It's fantastic to learn about your impressive
work.

I have previously worked on differentiating *Karenia brevis* and
Coccolithophores along the Florida Coast using pigment analysis and cell
counts for remote sensing. I've shared a paper below on observing *Karenia
brevis* from space, which might pique your interest.

https://www.frontiersin.org/articles/10.3389/fmars.2019.00523/full

**Reply by Philliec459** [reply]
Thank you so much for this valuable paper. We will study in detail. Would you mind if I sent this to others interested in Red Tide? 

Our involvement in Red Tide started with a talk from the owner of numerous restraunts in the Sarasota area where Red Tide has a huge negative effect on the economy. This led to our organization sponsoring a series of true Red Tide expert panel discussions, talks from Steve Davis of the Everglades Foundation and now NASA's interest and support. Your level of expertise is so welcome, and we look forward to all of us providing additional value on forecasting these HAB events. Thank you.

**Reply by bingqing-liu** [reply]
Please feel free to share and we are happy discuss to see if any collaborative work can be done in future with HyperCoast!

**Comment by Philliec459** [comment]
Thank you. One area that comes to mind is in the use of our STELLA-Q2 spectrometers to collect the hard data to create spectral libraries of vegetative species. This might not be directly applicable to HyperCoast until we actually start flying our drone conveyed STELLA instruments offshore to detect HAB that we can then compare to PACE data and any available HAB Cell Count data from NOAA. 

Along a similar line. we plan to study Mangrove Habitats and vegetative species similar to what was done by the Charlotte Harbor National Estuary Program(http://chnep.wateratlas.usf.edu/upload/documents/Mangrove-Heart-Attack-Draft-30Sept2016.pdf) in their Mangrove Habitat study. We are hoping to perform a similar study for Sarasota Bay as a part of our STEM initiatives.

We have not acquired any data yet, but we did create an example from about 200 STELLA readings with 16 different Test Patterns to give a flavor as to how we might approach this subject. Our python code and STELLA data can be made available to you if you like, but for now this pdf might serve as an example. At this time we are using transparent methods like Decision Tree logic, Spectral Angle and Knn to make straight forward predictions based on the mean end member data for each Test Pattern. However, we would like to consider other methods like spectral unmixing or other techniques that might be applicable. 

[Initial_Vegetative_Species_Library_Development_and_Applications.pdf](https://github.com/opengeos/HyperCoast/files/15355326/Initial_Vegetative_Species_Library_Development_and_Applications.pdf)

**Reply by bingqing-liu** [reply]
This is exciting! The Gulf offers tons of opportunities for studying blue carbon habitats, and I believe STELLA will play a significant role in researching ecosystems like mangroves, seagrass, and marshes. Please keep us updated on the spectral libraries—such resources would be invaluable for hyperspectral analysis. We're currently building a spectral library in our lab that includes both algae and aquatic plants, utilizing a hyperspectral micro and macro imaging system. If you're interested in the spectra of red tide species, I'd be happy to measure it in the lab. Let me know!

**Reply by Philliec459** [reply]
Measurements of Red Tide samples would be great. What other spectral libraries do you have? Collaboration. is key in this vast business. Our Science and Technology Society (STS) has held a number or events focused in Red Tide, but our Panel Discussions led by true experts in this area (Barbara Kirkpatrick, Dave Tomasko and Charlie Hunsicker) have been extremely good.  If we know more about your lab capabilities, then we might foster additional collaboration with these experts too. 

I am still reading your paper on Hurricane Impact on Phytoplankton. Very good. Will we ever be able to create a Figure 10 from PACE data? Dr. Wu has been most helpful is providing us the tools to now find and download PACE data, and now we want to be able to create our own Figure 10 from your great paper.

**Comment by Hamna607** [comment]
It's fascinating to see how the Sarasota Science and Technology Society (STS) is pushing the boundaries of spectral analysis and remote sensing with NASA's STELLA spectrometers. The integration of these advanced tools with Landsat data to improve NDVI calculations and the potential use of drones for PACE Red Tide interpretations are exciting developments.

In the realm of innovation and research, similar strides are being made in the Dutch construction sector by Stichting Bouwresearch. This organization plays a crucial role in advancing construction practices through rigorous research and technological development. Just as STS leverages cutting-edge technology for environmental monitoring, [Stichting Bouwresearch](https://startupio.com/stichting-bouwresearch/) focuses on pioneering sustainable building techniques and fostering technological advancements in construction. Their work exemplifies the kind of impactful research and development that can drive significant improvements in various fields.
